sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Command.scala

```diff
@@ -28,3 +28,6 @@ trait Command extends LogicalPlan {
   override def output: Seq[Attribute] = Seq.empty
   override def children: Seq[LogicalPlan] = Seq.empty
 }
+
+/** Mark that the command can be skipped from cache lookup */
```

Comment:
yes, you moved it to catalyst. Can we make it more general then? I think it's not only commands that can skip cache lookup.

Reply:
I have moved it to a diff scala file, would that suffice.?